datastore/src/test/java/io/spine/server/storage/datastore/DsAggregateStorageShould.java

```diff
@@ -98,6 +112,67 @@ public void fail_to_write_invalid_record() {
                             AggregateEventRecord.getDefaultInstance());
     }
 
+    @Test
+    public void set_limit_for_history_backward_query() {
+        final DsAggregateStorage<ProjectId> storage = (DsAggregateStorage<ProjectId>) getStorage();
+        final int batchSize = 10;
+        final AggregateReadRequest<ProjectId> request = new AggregateReadRequest<>(newId(),
+                                                                                   batchSize);
+        final EntityQuery historyBackwardQuery = storage.historyBackwardQuery(request);
+
+        final int queryLimit = historyBackwardQuery.getLimit();
+        assertEquals(batchSize, queryLimit);
+    }
+
+    @Test
+    public void allow_aggregate_loading_if_snapshot_trigger_was_decreased_in_AggregateRepository() {
```

Comment:
To emphasize those parts that matter, I would name it 
`still_load_aggregates_properly_after_snapshot_trigger_decrease_at_runtime`.